tests/Resources/ti.ui.webview.test.js

```diff
@@ -501,6 +502,47 @@ describe('Titanium.UI.WebView', function () {
 		win.open();
 	});
 
+	it.ios('blockedurl', function (finish) {
```

Comment:
Could this be ran on Android too?

Reply:
Unfortunately no. On Android, we can only "natively" block the links the end-user tapped on. We can't block the URLs that are loaded programmatically. This is something we document.

We can natively detect the URL being navigated to after the HTTP request has been sent by the `WebView` on Android, but then it's already too late. We may not be able to abort it in time. Especially if a cached HTTP response is received.